modules/openapi-generator/src/main/resources/erlang-server/handler.mustache

```diff
@@ -225,7 +225,7 @@ handle_request_json(
                 LogicHandler,
                 OperationID,
                 Req1,
-                Populated
+                maps:merge(State#state.context, Populated)
```

Comment:
Just to confirm. Shouldn't this be merge in the reverse order in order to keep the request specific values?
```suggestion
                maps:merge(Populated, State#state.context)
```

Reply:
https://erlang.org/doc/man/maps.html#merge-2

I was shooting for backwards compatibility, in such that a user is used to having only the parameters input into context. this change wouldn't overwrite any user input parameters and preserve any functionality where a user may accidentally overwrite with the addition of the context being passed from security definitions. So your suggestion would place user parameters in the context, then overwrite with the security definitions. Makes sense from a data transformation POV. I'll make the change later today.